helix-rest/src/main/java/org/apache/helix/rest/clusterMaintenanceService/MaintenanceManagementService.java

```diff
@@ -423,6 +422,23 @@ private StoppableCheck performCustomInstanceCheck(String clusterId, String insta
     return result;
   }
 
+  public static List<String> getListFromJsonPayload(JsonNode jsonContent) throws IOException {
+    if (jsonContent == null) {
+      return Collections.emptyList();
+    }
+    return OBJECT_MAPPER.convertValue(jsonContent, List.class);
+  }
+
+  public static Map<String, String> getMapFromJsonPayload(JsonNode jsonContent) throws IOException {
+    Map<String, String> result = new HashMap<>();
+    if (jsonContent == null) {
+      return result;
+    }
```

Comment:
You could simplify this logic with the use of a ternary operator, which would let you save the work of instantiating an empty HashMap instance.

Reply:
TFTR. Good suggestion. Updated.